domain/consensus/processes/blockvalidator/block_header_in_isolation.go

```diff
@@ -59,9 +59,22 @@ func (v *blockValidator) checkParentsLimit(header externalapi.BlockHeader) error
 }
 
 func (v *blockValidator) checkBlockVersion(header externalapi.BlockHeader) error {
-	if header.Version() != constants.BlockVersion {
-		return errors.Wrapf(
-			ruleerrors.ErrWrongBlockVersion, "The block version should be %d", constants.BlockVersion)
+	/*
+		if header.Version() != constants.BlockVersion {
+			return errors.Wrapf(
+				ruleerrors.ErrWrongBlockVersion, "The block version should be %d", constants.BlockVersion)
+		}
+	*/
```

Comment:
Is there a reason why this is commented? Explicit error in case an old node would send a wrong version block after hardfork is needed.

Reply:
It has been replaced by a conditional check on BlockVersionAfterHF BlockVersionBeforeHF to reject block with bad version after or before HF